tests/vm_backup_test.go

```diff
@@ -45,138 +65,121 @@ var _ = Describe("[smoke] VM Backup", func() {
-
-			By("Deleting VM")
-			err = DeleteVirtualMachine(*kvClient, namespace.Name, vm.Name)
-			Expect(err).ToNot(HaveOccurred())
-
-			By("Creating restore")
-			err = CreateRestoreForBackup(timeout, "test-backup", "test-restore", r.BackupNamespace, true)
-			Expect(err).ToNot(HaveOccurred())
-
-			rPhase, err := GetRestorePhase(timeout, "test-restore", r.BackupNamespace)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(rPhase).To(Equal(velerov1api.RestorePhaseCompleted))
-
-			By("Verifying VM")
-			err = WaitForVirtualMachineStatus(*kvClient, namespace.Name, vm.Name, kvv1.VirtualMachineStatusStopped)
-			Expect(err).ToNot(HaveOccurred())
-		})
-
-		It("started VM should be restored", func() {
-			err := WaitForVirtualMachineStatus(*kvClient, namespace.Name, vm.Name, kvv1.VirtualMachineStatusRunning)
-			Expect(err).ToNot(HaveOccurred())
-			ok, err := WaitForVirtualMachineInstanceCondition(*kvClient, namespace.Name, vm.Name, kvv1.VirtualMachineInstanceAgentConnected)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(ok).To(BeTrue(), "VirtualMachineInstanceAgentConnected should be true")
-
-			By("Creating backup")
-			err = CreateBackupForNamespace(timeout, "test-backup", namespace.Name, snapshotLocation, r.BackupNamespace, true)
-			Expect(err).ToNot(HaveOccurred())
-
-			phase, err := GetBackupPhase(timeout, "test-backup", r.BackupNamespace)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(phase).To(Equal(velerov1api.BackupPhaseCompleted))
-
-			By("Stopping VM")
-			err = StopVirtualMachine(*kvClient, namespace.Name, vm.Name)
-			Expect(err).ToNot(HaveOccurred())
-			err = WaitForVirtualMachineStatus(*kvClient, namespace.Name, vm.Name, kvv1.VirtualMachineStatusStopped)
-			Expect(err).ToNot(HaveOccurred())
-
-			By("Deleting VM")
-			err = DeleteVirtualMachine(*kvClient, namespace.Name, vm.Name)
-			Expect(err).ToNot(HaveOccurred())
-
-			By("Creating restore")
-			err = CreateRestoreForBackup(timeout, "test-backup", "test-restore", r.BackupNamespace, true)
-			Expect(err).ToNot(HaveOccurred())
-
-			rPhase, err := GetRestorePhase(timeout, "test-restore", r.BackupNamespace)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(rPhase).To(Equal(velerov1api.RestorePhaseCompleted))
-
-			By("Verifying VM")
-			err = WaitForVirtualMachineStatus(*kvClient, namespace.Name, vm.Name, kvv1.VirtualMachineStatusRunning)
-			Expect(err).ToNot(HaveOccurred())
-		})
+	It("Backing up stopped VM should succeed", func() {
+		By("Stoppin a VM")
+		err := StopVirtualMachine(*kvClient, namespace.Name, vm.Name)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Creating backup")
+		err = CreateBackupForNamespace(timeout, backupName, namespace.Name, snapshotLocation, r.BackupNamespace, true)
+		Expect(err).ToNot(HaveOccurred())
+
+		phase, err := GetBackupPhase(timeout, backupName, r.BackupNamespace)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(phase).To(Equal(velerov1api.BackupPhaseCompleted))
+	})
+
+	It("Backing up started VM should succeed", func() {
```

Comment:
Is this not the exact same thing as the start of`"started VM should be restored"` maybe combine the two?

Reply:
yes